code/datums/diseases/advance/symptoms/sneeze.dm

```diff
@@ -35,5 +36,13 @@ Bonus
 				M.emote("sniff")
 			else
 				M.emote("sneeze")
-				A.spread(A.holder, 5)
-	return
\ No newline at end of file
+				Spreadzone(M, A)
+				var/obj/item/I = M.get_active_hand()
+				if(I && I.w_class > 0)

```

Comment:
Eh, It's a little hackey way to ensure all items are dropped. I didn't want to do Wclass > 1 because it'll allow you to hold pens (I think). Is this fine or...?